packages/components/pie-toggle-switch/src/toggle-switch.scss

```diff
@@ -1,3 +1,113 @@
-// TODO - please remove the stylelint rule above when SCSS is added to this file
-/* stylelint-disable no-empty-source */
+@mixin toggle-switch-transition($property) {
+    @media (prefers-reduced-motion: no-preference) {
+        transition: $property 0.15s cubic-bezier(0.4, 0, 0.9, 1) 0s;
+    }
+}
 
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+.c-toggleSwitch {
+    --toggle-switch-colour: var(--dt-color-interactive-form);
+    --toggle-switch-colour--checked: var(--dt-color-interactive-brand);
+    --toggle-switch-colour--disabled: var(--dt-color-disabled-01);
```

Comment:
```suggestion
    --toggle-switch-bgColor: var(--dt-color-interactive-form);
    --toggle-switch-bgColor--checked: var(--dt-color-interactive-brand);
    --toggle-switch-bgColor--disabled: var(--dt-color-disabled-01);
```

Reply:
I've had to use kebab case as the linter throws errors with camel case here. @ashleynolan